### ISSUE TITLE
An occasional high-risk problem 

### ISSUE THREAD

**Issue by zhrf2020** [fedora-sysv/chkconfig#83]
Source file: `chkconfig.c` where the function: qsort(services, numServices, sizeof(*services), serviceNameCmp); here the function serviceNameCmp

```c
static int serviceNameCmp(const void *a, const void *b) {
     return strcmp(*(char **)a, *(char **)b);
}
```

may cause function qsort to segfault
Reference: https://stackoverflow.com/questions/33774013/getting-segmentation-fault-when-using-qsort-for-sorting-array-of-strings-in-c

**Comment by lnykryn** [comment]
https://github.com/fedora-sysv/chkconfig/pull/87

**Comment by jamacku** [comment]
Fixed via #87. Thank you for reporting this issue.
